src/components/OptionsAccordion/OptionsAccordion.stories.tsx

```diff
@@ -2,58 +2,65 @@ import React, { useState } from 'react'
 
 import { StoryObj } from '@storybook/react'
 
-import OptionsAccordion from '.'
+import { OptionsAccordion, OptionsType, PanelOptions } from '.'
 
 export default {
   title: 'OptionsAccordion',
   component: OptionsAccordion
 }
 
-type OptionsType = { [key: string]: boolean }
-const Genre: OptionsType = {
-  Action: false,
-  Adventure: false,
-  'Role-Playing': false,
-  Strategy: false,
-  Simulation: false,
-  Sports: false,
-  Racing: false
+const Genre: PanelOptions = {
+  Action: { selected: false },
+  Adventure: { selected: false },
+  'Role-Playing': { selected: false },
+  Strategy: { selected: false },
+  Simulation: { selected: false },
+  Sports: { selected: false },
+  Racing: { selected: false }
 }
 
-const Systems: OptionsType = {
-  Windows: false,
-  Mac: false,
-  Linux: false,
-  Browser: false
+const Systems: PanelOptions = {
+  Windows: { selected: false },
+  Mac: { selected: false },
+  Linux: { selected: false },
+  Browser: { selected: false }
 }
 
-const Version: OptionsType = {
-  Alpha: false,
-  Beta: false,
-  Stable: false
+const Version: PanelOptions = {
+  Alpha: { selected: false },
+  Beta: { selected: false },
+  Stable: { selected: false }
 }
 
-const Others: OptionsType = {
-  'Token required': false,
-  Downloaded: false,
-  'Show hidden': false,
-  'Show non-available': false
+const Others: PanelOptions = {
+  'Token required': { selected: false },
+  Downloaded: { selected: false },
+  'Show hidden': { selected: false },
+  'Show non-available': { selected: false }
 }
 
-const Scrollable: OptionsType = Object.fromEntries(
-  Array.from({ length: 25 }, (_, i) => [`Item${i + 1}`, false])
+const Chains: PanelOptions = {
+  1: { selected: false, displayName: 'Ethereum Mainnet' },
+  2: { selected: false, displayName: 'Some Other Chain' },
+  1010101: { selected: false, displayName: 'Another Chain' },
+  1231123: { selected: false }
+}
```

Comment:
Couldn't you map the chains to their display name and use them as the key? That way we can keep the current API and not need the `selected` property?

Reply:
Hmm not sure if I understand. The key I think we need to use is the chainId (and caip 2 https://chainagnostic.org/CAIPs/caip-2 once we have non-evm tokens), but the previous implementation only used the key as the display value.

Are you proposing we just have a separate map prop for chainId -> displayName? I'd be fine with that